duo_openvpn.py

```diff
@@ -235,9 +235,11 @@ def auth(self):
 def main():
 	username_trusted = True
 	username = os.environ.get('common_name')
+# this is the username as provided by the user - it cannot be trusted
+# we use it in case common_name is not present, or as password for OTP
+	username_untrusted = os.environ.get('username')
 	if (username == None):
-# take username as provided by the user - it cannot be trusted
-		username = os.environ.get('username')
+		username_untrusted = os.environ.get('username')
```

Comment:
I think this line should be:
`username_untrusted = os.environ.get('common_name')`
...or even:
`username_untrusted = username`

Since as-is, it's just re-assigning the same value to `username_untrusted` as was set prior to the conditional, so is redundant.